### PR TITLE
docs: update Sfyra docs

### PR DESCRIPTION
**Pull request by smira**
Some changes related to merge with Sidero project.

Signed-off-by: Andrey Smirnov <smirnov.andrey@gmail.com>

### CODE REVIEW

File: sfyra/README.md [hunk 1]
```diff
@@ -4,16 +4,15 @@ Integration test for Sidero/Arges.
 
 ## Running
 
-Build the test binary:
+Build the test binary and Sidero, push images:
 
-    make sfyra
+    make USERNAME=<username> PUSH=true
```
Comment (Ulexus) [inline sfyra/README.md:9]: Is the function of the USERNAME environment variable explained somewhere in the rest of the documentation?

**Review by Ulexus — APPROVED** [review]

File: sfyra/README.md (hunk 1 above)
Comment (smira) [reply sfyra/README.md:9]: it's kind of convention across all our repos, image is `$(REGISTRY)/$(USERNAME)/<imagename>`

**Review by andrewrynhard — APPROVED** [review]

**Pull request merged**